tools/snp.sh

```diff
@@ -904,13 +914,168 @@ attest_guest() {
     || { >&2 echo -e "FAIL: measurements do not match"; return 1; }
 }
 
+# Additional Functions Added here
+rhel_subscription_mgr_set_login(){
+  echo "Enter RedHat subscription Manager credentials"
+  read -p "Username: " RHEL_SUBS_MGR_USER
+  read -sp "Password: " RHEL_SUBS_MGR_PASS
+}
```

Comment:
Can we add a check to see if the subscription manager status is already active and only prompt if not active? Can you check out this example and test it out?
https://access.redhat.com/discussions/2217891

Also, only prompt for values if not already exported in the variables.